lib/api-client.js

```diff
@@ -32,7 +32,7 @@ SlackAPIClient.prototype.SlackError = SlackError;
  * @param {Function} [callback] An optional callback to propagate the response to.
  * @callback {[Error, ResponseBody, Response]}
  */
-SlackAPIClient.prototype.api = function(method, options, callback) {
```

Comment:
Will give more feedback / ideas over the next week. Roll with it :)

ᐧ

On Mon, Jun 15, 2015 at 10:16 AM, Fred K. Schott <notifications@github.com>
wrote:

> In lib/api-client.js
> <https://github.com/FredKSchott/slackey/pull/9#discussion_r32443528>:
>
> > @@ -32,7 +32,7 @@ SlackAPIClient.prototype.SlackError = SlackError;
> >   * @param {Function} [callback] An optional callback to propagate the response to.
> >   * @callback {[Error, ResponseBody, Response]}
> >   */
> > -SlackAPIClient.prototype.api = function(method, options, callback) {
>
> Ah wait no, I forgot call is already taken by JavaScript
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call
>
> .send() is clear enough.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FredKSchott/slackey/pull/9/files#r32443528>.
>



-- 
*Jonathon Kresner*
*Co-founder, CEO*
@AirPair <http://twitter.com/airpair> | *airpair.com
<http://www.airpair.com>*
@hackerpreneur <http://twitter.com/hackerpreneur>